test/uk/gov/hmrc/economiccrimelevyreturns/controllers/StartControllerSpec.scala

```diff
@@ -27,20 +29,24 @@ class StartControllerSpec extends SpecBase {
 
   val view: StartView = app.injector.instanceOf[StartView]
 
-  val controller = new StartController(
-    mcc,
-    fakeAuthorisedAction,
-    fakeDataRetrievalAction(),
-    view
-  )
+  class TestContext(eclReturnData: EclReturn) {
```

Comment:
Name parameter edlReturn rather than eclReturnData (to be consistent with elsewhere)?